capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -17,8 +17,8 @@
 public final class BookList {
 
   private Long id;
-  private String bookshelfID;
   private Long userID;
+  private String bookshelfID = "";
   private ArrayList<Long> collaboratorsIDs;
```

Comment:
I think it might be better to make this a HashSet rather than an ArrayList. We should also discuss if we want to make everything implement the Collection interface, so we have consistent style.